op-supervisor/supervisor/backend/db/db_test.go

```diff
@@ -170,9 +171,9 @@ func setupStubbedForUpdateHeads(chainID types.ChainID) (*stubLogDB, *stubChecker
 	// the checkpoint starts somewhere behind the last known cross-safe head
 	checkpoint := entrydb.EntryIdx(15)
 	// the last known cross-safe head is at 20
-	cross := entrydb.EntryIdx(20)
+	cross := heads.HeadPointer{LastSealedBlockNum: 20}
 	// the local head (the limit of the update) is at 40
-	local := entrydb.EntryIdx(40)
+	local := heads.HeadPointer{LastSealedBlockNum: 40}
```

Comment:
This change satisfied compilation errors for the tests but these tests are not meaningfully fixed yet.